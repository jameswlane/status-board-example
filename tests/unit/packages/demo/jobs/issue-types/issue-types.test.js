const fileToTest = require('../../../../../../packages/demo/jobs/issue-types/issue-types');

describe('Issue Types', function () {
  test('onRun to match snapshot', function () {
    expect(fileToTest.onRun).toMatchSnapshot();
  });
});