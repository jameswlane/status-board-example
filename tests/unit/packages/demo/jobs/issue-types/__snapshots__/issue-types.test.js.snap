// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Issue Types onRun to match snapshot 1`] = `[Function]`;
